doc/releases/mimic.rst

```diff
@@ -1,3 +1,23 @@
+v13.2.8 Mimic
+=============
+
+This is the fourth release in the Ceph Nautilus stable release series. Its sole
+purpose is to fix a regression that found its way into the previous release.
+
+Notable Changes
+---------------
+
+* Due to a missed backport, clusters in the process of being upgraded from
+  13.2.6 to 13.2.7 might suffer an OSD crash in build_incremental_map_msg.
+  This regression was reported in https://tracker.ceph.com/issues/43106
+  and is fixed in 13.2.8 (this release).
```

Comment:
You could add a note for users to directly upgrade to 13.2.8 from 13.2.6 to avoid this.